src/bundle.c

```diff
@@ -65,8 +65,9 @@ int list_installable_bundles()
 	while (list) {
 		file = list->data;
 		list = list->next;
-		printf("%s\n", file->filename);
+		printf("%s%s\n", file->is_experimental ? "*" : "", file->filename);
```

Comment:
instead of the ambigous `*` with the legend below why not just do this inline:
`bundlename (experimental)`

Reply:
@matthewrsj sounds good to me, I was using the * because that is what it was on #676 but I like it better the way you are suggesting it

Reply:
I'll make the change

Reply:
It's a lot better with the " (experimental)" string